components/DayPicker.tsx

```diff
@@ -0,0 +1,39 @@
+import Helmet from 'react-helmet'; // Date picker inline styling for select range functionality
+import ReactDayPicker, { DayPickerProps } from 'react-day-picker';
+import 'react-day-picker/lib/style.css'; // Date picker styling
+
+function DayPickerStyling() {
+  return (
+    <Helmet>
+      <style>{`
+        .Selectable .DayPicker-Day--selected:not(.DayPicker-Day--start):not(.DayPicker-Day--end):not(.DayPicker-Day--outside) {
+          background-color: #f0f8ff !important;
+          color: #4a90e2;
+        }
+        .Selectable .DayPicker-Day {
+          border-radius: 0 !important;
+        }
+        .Selectable .DayPicker-Day--start {
+          border-top-left-radius: 50% !important;
+          border-bottom-left-radius: 50% !important;
+        }
+        .Selectable .DayPicker-Day--end {
+          border-top-right-radius: 50% !important;
+          border-bottom-right-radius: 50% !important;
+        }
+      `}</style>
+    </Helmet>
+  );
+}
+
+/**
+ * DayPicker component from {@link http://react-day-picker.js.org/| react-day-picker} library with additional styling.
+ */
+export default function DayPicker(props: DayPickerProps) {
```

Comment:
Given the purposes and use cases for the DayPicker of our team, I think we can abstract this a bit further. For instance, all our day picker components will have the same styling, and we just need to define the `modifiers` and the `onDayClick` props. We don't actually need to pass the `{ start: range.from || undefined, end: range.to || undefined }` value to the `DayPicker` component - we could just abstract this and have our `DayPicker` component take a `range` prop which is an object.

So our props for the `DayPicker` component could look something like this:

```typescript
type DayPickerProps {
  readonly numberOfMonths?: number; // # of months, default to 1
  readonly range: RangeModifier;
  readonly onDateChange: (day: <whatever the type is>) => void; // Handle date change
}
```

This way, we can abstract most of the functionality in the `DayPicker` component in this file. We can also even abstract our `range` state by writing our own custom hook `useDayPicker`. This will allow us to also hide away functions such as `dateRangeString()` below, so that we don't repeat that logic in every module in which we use the `DayPicker` component.

Would be happy to sync up with you @EmilioMena on this when I'm done my finals!

Reply:
Good point! I tried address this in my last commit, lmk if i can add anything else